File: year2020/month10/day1009/ms_1009.cs
using System;

namespace algorithm_75
{
    class Program
    {
        static void OddEven(int n) 
        {
            if(n <= 0)
                return;

            OddEven(n-2);
            Console.Write($"{n} ");
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(0 < n)
                OddEven(n);
        }
    }
}

File: year2020/month10/day1009/ms_1009.cpp
#include <iostream>

using namespace std;

void OddEven(int n);

int main() {
	int n;

	cin >> n;

	if (0 < n)
		OddEven(n);

	return 0;
}

void OddEven(int n) {
	if (n <= 0)
		return;

	OddEven(n - 2);
	cout << n << ' ';
}